L4/README.md

```diff
@@ -0,0 +1,11 @@
+# L4 setup
+
+The setup in this folder was created for gotm v5. Running a later version of gotm you will need to update the `gotm.yaml` file. This can be done by following the description below.
+
+Option 1:
+1. Download the `gotm` updater from [here](https://github.com/gotm-model/code/blob/master/scripts/python/update_setup.py)
```

Comment:
I couldn't actually figure out how to just download one file from github. The majority of people should already have the script in their gotm code file.  Prehaps an alternative is to say  run “python3 <GOTM_CODE_DIR>/scripts/python/update_setup.py gotm.yaml --gotm <PATH_TO_GOTM_EXECUTABLE>

Reply:
Ah yes, that is a good point. I will add that in.